src/main/kotlin/skill/TaoQu.kt

```diff
@@ -123,13 +125,29 @@ class TaoQu : MainPhaseSkill() {
             }
             if (r is RobotPlayer) {
                 GameExecutor.post(g, {
-                    val playerAndCard = g.players.flatMap {
-                        if (!it!!.alive || !it.isEnemy(r)) emptyList()
-                        else it.messageCards.mapNotNull { card ->
-                            if (!card.colors.any { c -> c in colors }) null
-                            else PlayerAndCard(it, card)
+                    val color = listOf(Red, Blue, Black).filter {
+                        r.cards.count(it) >= 2
+                    } // 手牌中出现两次的颜色
+                    val movecards = ArrayList<Card>()
+                    val playerAndCard: PlayerAndCard?
+                    var value = Int.MIN_VALUE
+                    for (p in g.players) {
+                        p!!.alive || continue // 死亡玩家跳过
+                        for (movecard in p.messageCards) {
```

Comment:
136行遍历的时候，138行还移除，会报`java.util.ConcurrentModificationException`
```suggestion
                        for (movecard in p.messageCards.toList()) {
```